### ISSUE TITLE
Can the gemtc package produce the predictive interval for network meta-analysis?

### ISSUE THREAD

**Issue by zddzxxsmile** [gertvv/gemtc#54]
Dear gertvv, 

Thanks for your great contribution.

Can the gemtc package produce the predictive interval for network meta-analysis?

**Comment by gertvv** [comment]
No, this is not supported out of the box. You should be able to calculate it using the posteriors. This would be a good addition so tagging as a possible enhancement. No plans to work on it any time soon, but a pull request is more than welcome.
